### ISSUE TITLE
Uniforms, playing cti (crCTI, wasp ctiRHS)

### ISSUE THREAD

**Issue by dvdbrewster** [acemod/ACE3#2379]
Using ACE3, playing proman public (crCTI-24), and having cba, Squad leaders are spawned w/o uniforms. (crCTI) and WASP rhs (cti@40) any AI that is same as your profile avatar by name, 

CBA_A3
AllInArmATerrainOack
ACE 3
RHS (east/west)
ma0_bloodmist_a3

addon  aceserver pbo
no placed modules

the issue is on dedicated server , playing cti missions. crCTI proman public(v 1.22)
and WASP warfare v 2.15. 
 The  ais spawn w/o uniforms in both of these missions. It doesnt do that on other server running 
CBA _A3
AiA terrainPack (latest),  
lhm glasses
mao_blood_mist,

I tried changing some parameters with server cfg(server pbo) Still ais are undressed.
U will say this is sti scripts with their ai, but mayB u have heard of this b4 and somebody changed anything to correct it.?, MayB placed a module , tho  cti is dynamic respawn.

U can reproduce this by launching the game, as above.

This happens onn dedicated server.

If u insist Ill get a rpt file, however since spawning in cti due to upgrades and base movement, I have -no logs- as this creates a very large file  when playing, And since this is common to WASP and crCTI, but doesnt seem to affect beCTI, u could direct my attention.

**Comment by jonpas** [comment]
Please follow http://ace3mod.com/wiki/user/how-to-report-an-issue.html

**Comment by alganthe** [comment]
mission related issue please report back to the mission maker not us.

**Comment by dvdbrewster** [comment]
@alganthe  WarLords is doing the same thing, Seems squad leaders are mosty affected,  that's 3 missions

**Comment by Glowbal** [comment]
This sounds like it's an issue with those missions, not ACE. Please follow the instructions as posted by @jonpas 

Try to see if you can reproduce this issue with just CBA and ACE3. It's possible that it is an issue with the uniforms you are trying to give to the AI.

**Comment by dvdbrewster** [comment]
I did it. Ran CBA and ACE3.  Still theres naked ais.  Seems its from Officers spawning, I tried removing the prptection pbo files, even. There was no difference.   It might work itself out in the future.  Proman crcti all the squad leaders spawn w/o uniform. (agm they spawn dressed) I couldn't try the wasp w/o rhs, 
as this is the only vers I have, (rhs).  Seems mayB both rhs and crCTI proman are porting from A2. I might be wrong.   AnyWay thanks.
